static/app.css

```diff
@@ -0,0 +1,499 @@
+	padding: 5px;
+	color: #3c9076;
+	text-align: center;
+}
+.topEarnersHead {
+	margin: 0 auto;
+	color: inherit;
+	font-weight: bold;
+	text-transform: uppercase;
+}
+.topEarner {
+	display:inline-block;
+	width: 220px;
+	margin: 10px;
+	padding: 5px;
+	border: solid #3c9076;
+	border-width: 0 2px;
+	border-radius: 10px;
+	font-size: 18px;
+	text-transform: uppercase;
+}
+.topEarner:hover {
+	background-color: #9ee6d0;
+}
+.topEarner a {
+	color:inherit;
+	text-decoration: none;
+	cursor: pointer;
+}
+.topEarner p {
+	margin: 0 auto;
+}
+.topEarner p:first-child {
+	font-weight: bold;
+}
+div#share-tools{
+	color:white;
+	float:right;
+	position:relative;
+	font-size:22px;
+}
+div#share-tools:hover {
+	cursor: pointer;
+}
+div#share-tools span:last-child{
+	margin-right:4px;
+	margin-left: 4px;
+}
+
+/* Keep our tophat css from blowing out the Tribune menu */
+/* ----------------------------------------------------- */
+
+.trb_3pt_nav li {
+        margin-bottom: inherit;
+}
+.trb_3pt_mainLink_a {
+	color: white!important;
+}
+
+/* This document is for project-specific css only. */
+
+hr {
+	border-color: white;
+}
+div.sharethiswrapper {
+    margin: auto;
+    text-align: center;
+    width: 200px;
+}
+div.sharethiswrapper a img {
+    float: none;
+    margin: auto;
+}
+
+.highlight{
+    background-color:#ffffff!important;
+}
+div.for5  {
+	border:1px solid #ffffff;
+}
+#dbform {
+	text-align: center;
+}
+div.rangeselector, div.dbsearch {
+	position: relative;
+	display: inline-block;
+	width:260px; 
+	/*height:120px;*/
+	height:auto;
+	padding: 15px;
+    margin: 10px;
+    border-radius: 8px;
+    background-color:#3c9076;
+    color: white;
+    font-weight: bold;
+    vertical-align: top;
+}
+div.roundcaveat{
+	color:#9ee6d0;
+}
+div.dbsearch input[type="text"], div.dbsearch input[type="search"], div.rangeselector input[type="text"], div.rangeselector input[type="search"]{ 
+    width:98%;
+    padding: 5px 0 5px 5px;
+    margin-bottom: 10px;
+    border: none;
+    border-radius: 5px;
+}
+div.dbsearch .label{
+	min-height:30px;
+}
+td {
+	padding: 5px;
+}
+td.fldtypetwo {
+	display:none;
+}
+table#dbresults{
+	font-size:12px;
+	margin: 20px auto;
+	width:100%;
+}
+#dbresults tr.fancyfloat:hover{
+	background:#9ee6d0;
+	cursor:pointer;
+}
+table#dbresults tr td span{
+	margin:6px 3px 6px 3px;
+	display:block;
+}
+#dbresults tr.odd{
+	/*background: #c5f6fa;*/
+}
+#dbresults tr.even, #dbresults thead{
+	background: #ccf3e7
+}
+#dbresults thead {
+	font-weight: 900;
+}
+#verkapselung{
+	max-width: 1000px;
+	margin:0 auto;
+	float:none;
+	padding: 20px; 
+	text-align: left;
+}
+@media (max-width: 768px) {
+	div#verkapselung {
+		padding: 10px; 
+	} 
+}
+div.realmin, div.realmax{
+	display:none;
+}
+td.sortingbyme{
+	/*A subtle, relatively succinct cross-platform solution for opacity, as described at  http://robertnyman.com/2010/01/11/css-background-transparency-without-affecting-child-elements-through-rgba-and-filters/ */
```

Comment:
Probably this hack is no longer needed. I don't think we should support anything below IE11.

Reply:
took this out